src/components/ActivityDescription.tsx

```diff
@@ -32,7 +32,7 @@ function TypeSpan(props: { flag?: string; title: string }) {
       <Image
         alt={title}
         boxSize="1em"
-        src={`img/${flag}.gif`}
+        src={flagURLs[flag]}
```

Comment:
We can get the old static links back if needed but this is the way Parcel suggests because it works better w.r.t. caching.